### PR TITLE
Add metric owner to metric docs

### PR DESCRIPTION
**Pull request by nstokoe**
CleanShot 2023-09-27 at 17 04 11@2x

Open to suggestions for how best to phrase this.

### CODE REVIEW

**Comment by davidsonsarah** [comment]
We could add an example like: "In some organizations the owner assigned to the metric may be the functional person, i.e. VP of Sales is responsible for average time-to-close metric. In other orgs, the owner assigned may be the person responsible for modeling and managing the metric data, i.e. a sales analyst."

**Comment by nstokoe** [comment]
CleanShot 2023-09-29 at 10 12 11

@davidsonsarah thanks for the suggestion. I've updated the text with the gist of your words.

File: pages/docs/metrics.mdx [hunk 1]
```diff
@@ -66,7 +66,10 @@ Once you have a data model, you can create a metric by following these steps:
    - **Description** - A description of the metric.
    - **Labels** - Labels are used to categorize or tag similar metrics.
      For example, you might have a label for each department in your organization. Metrics are also filterable by label on the metrics page.
-   - **Owner** - The owner of the metric. This is the person responsible for the performance of the metric.
+   - **Owner** - The person responsible for overseeing and managing the metric is the metric owner. 
+     The specific responsibilities of the metric owner can vary depending on the needs and structure of your organization or team. 
+     For example, in some organizations, the owner may hold a functional role like a VP of Sales and be responsible for high-level sales metrics such as average time-to-close. 
+     In other organizations, the assigned metric owner may be a sales analyst who is responsible for modeling and managing the metric data."
```
Comment (annanixon) [inline pages/docs/metrics.mdx:72]: nit: there's an extra " at the end of the sentence

**Review by annanixon — APPROVED** [review]
🚀

**Pull request merged**